htdocs/product/list.php

```diff
@@ -409,7 +411,10 @@
 
 $sql = 'SELECT p.rowid, p.ref, p.label, p.fk_product_type, p.barcode, p.price, p.tva_tx, p.price_ttc, p.price_base_type, p.entity,';
 $sql .= ' p.fk_product_type, p.duration, p.finished, p.tosell, p.tobuy, p.seuil_stock_alerte, p.desiredstock,';
-$sql .= ' p.tobatch,';
+$sql .= ' p.tobatch, ';
+if (isModEnabled('workstation')) {
+	$sql .= ' p.fk_default_workstation, ws.status as status_workstation, ws.ref as ref_workstation, ';
```

Comment:
Do you set your mysql config to "strict" ? See https://wiki.dolibarr.org/index.php?title=Langages_et_normes#Sp.C3.A9cificit.C3.A9s_MySQL
I think if you are set in this recommended mode, you should get a SQL error with this change as the new fields are not inside the group by